src/components/show-products/index.js

```diff
@@ -43,19 +67,34 @@ export const ShowProducts = () => {
 
   return (
     <ContentContainer>
+      <form className="filter-form">
+        <label htmlFor="filter">
+          Filter items
+          <input
+            className="input-item"
+            type="search"
+            id="filter"
+            name="q"
+            aria-label="Search through list content"
+            placeholder="Start typing here..."
+            value={searchTerm}
+            onChange={(event) => setSearchTerm(event.target.value)}
+          />
+        </label>
+      </form>
       <ul>
-        {products?.map(({ id, productName, lastPurchaseDate, timeFrame }) => (
+        {list.map(({ id, productName, lastPurchaseDate, timeFrame }) => (
           <li className="checkbox-item" key={id}>
             <label htmlFor={id} className="checkbox-label">
               <input
                 type="checkbox"
                 id={id}
                 name={productName}
-                onChange={() => handleOnChange(id)}
+                onChange={() => handleCheckboxChange(id)}
                 checked={
                   lastPurchaseDate && compareDates(lastPurchaseDate.toDate())
                 }
-                ariaLabel={timeFrames[timeFrame]}
+                aria-label={TimeFrames[timeFrame]}
```

Comment:
Nice! I recently read somewhere that aria attributes are not camelCased in JSX. I'll try to search this out so you could read it too!

Reply:
I read it in the new [Beta React Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) it is wooooorth the read! 

I loved this new docs, as they're more functional components oriented.